importlib_metadata/_meta.py

```diff
@@ -65,3 +65,5 @@ def read_text(self, encoding=None) -> str: ...  # pragma: no cover
     def read_bytes(self) -> bytes: ...  # pragma: no cover
 
     def exists(self) -> bool: ...  # pragma: no cover
+
+    def resolve(self) -> bool: ...  # pragma: no cover
```

Comment:
Why was this added?

Reply:
Not only does this change seem unnecessary, it's also incorrect. `.resolve` doesn't return a `bool`. It returns another `SimplePath`.

Moreover, I suspect other implementations of `SimplePath` don't implement resolve.

In particular, the [zipfile.Path](https://docs.python.org/3/library/zipfile.html#path-objects) is meant to be another implementation of `SimplePath` and is used by `PathDistribution` to support packages in a zip file. It also looks like `zipfile.Path` doesn't support `.relative_to` or `.as_posix`, so adding `.resolve()` to the protocol won't be sufficient.

It does appear that the `_read_files_egginfo_installed` routine has never worked for zipfile.Paths.

And if nobody's asking for it to do so, I'm not going to worry about it. Eggs are deprecated and support is best-effort.

Reply:
I've found that removing this change allows the tests to pass, so I'm going to assume this change is not needed and proceed without it. Removed in e4d1dcca72.